test/everypolitician/popolo/organization_test.rb

```diff
@@ -33,4 +33,13 @@ def test_organizations_subtraction
     just_org_1 = Everypolitician::Popolo::Organizations.new([org1])
     assert_equal [Everypolitician::Popolo::Organization.new(org2)], all_orgs - just_org_1
   end
+
+  def test_organization_wikidata
+    org = Everypolitician::Popolo::Organization.new(
+      id: 'abc', 
+      name: 'ACME', 
+      identifiers: [{identifier: "Q288523", scheme: "wikidata"}],
+    )
+    assert_equal org.wikidata, 'Q288523'
```

Comment:
The order of the arguments for `assert_equal` in Ruby is `(expected, actual)`, so while this will work it will give slightly confusing output if the test were to fail:

```
> assert_equal org.wikidata, 'asdf'

Expected: "Q288523"
  Actual: "asdf"
```